crates/swift-integration-tests/src/option.rs

```diff
@@ -35,10 +35,16 @@ mod ffi {
 
     extern "Rust" {
         type OptTestOpaqueRustType;
+        type OptTestOpaqueRefRustType;
 
         #[swift_bridge(init)]
+        #[swift_bridge(associated_to = OptTestOpaqueRustType)]
```

Comment:
Need to remove this `associated_to`. I'll open an issue about emitting a compile time error t `#[swift_bridge(init)]` and `#[swift_bridge(associated_to = OptTestOpaqueRustType)]` from both being used.

Removing `associated_to` should fix the problem.

Reply:
Done - and all passing locally for me now :)

Reply:
> I'll open an issue

Nevermind the real problem is that if there are two `#[swift_bridge(..)]` attributes the first one gets ignored.

I'll still track it. Thanks.